addons/event/views/event_views.xml

```diff
@@ -6,6 +6,79 @@
         <menuitem name="Events" id="event_main_menu" sequence="65" groups="event.group_event_user" web_icon="event,static/description/icon.png"/>
         <menuitem parent="event_main_menu" id="menu_reporting_events" sequence="99" groups="event.group_event_manager" name="Reporting"/>
 
+        <!-- EVENT.CATEGORY VIEWS -->
+
+        <record model="ir.ui.view" id="view_event_category_tree">
+            <field name="name">event.category.tree</field>
+            <field name="model">event.category</field>
+            <field name="arch" type="xml">
+                <tree string="Event Category">
+                    <field name="sequence" widget="handle"/>
+                    <field name="name"/>
+                    <field name="tag_ids" widget="many2many_tags" />
+                </tree>
+            </field>
+        </record>
+
+        <record model="ir.ui.view" id="view_event_category_form">
+            <field name="name">event.category.form</field>
+            <field name="model">event.category</field>
+            <field name="arch" type="xml">
+                <form string="Event Category">
+                    <sheet>
+                        <div class="oe_title">
+                            <label for="name" class="oe_edit_only"/>
+                            <h1><field name="name"/></h1>
+                        </div>
+                        <group>
+                            <field nolabel="0" string="sequence" name="sequence"/>
+                            <field name="tag_ids" context="{'default_category_id': active_id}">
+                                <tree string="Tags" editable="bottom">
+                                    <field name="sequence" widget="handle"/>
+                                    <field name="name"/>
+                                </tree>
+                            </field>
```

Comment:
Make a many2many_tags widget instead and override tags "name_get" to show Category: Tag